controller/Makefile

```diff
@@ -50,6 +50,14 @@ deploy: build dev-release restart
 runserver:
 	python manage.py runserver
 
+postgres:
+	docker start postgres || docker run --restart="always" -d -p 5432:5432 --name postgres postgres:9.4.1
+	docker exec postgres createdb -U postgres deis 2>/dev/null || true
+	@echo "To use postgres for local development:"
+	@echo "    export PGHOST=127.0.0.1"
```

Comment:
127.0.0.1 doesn't work for `boot2docker` unfortunately: ```OperationalError: could not connect to server: Connection refused``` when running the controller unit tests.

Maybe change it to:
```sh
@echo "    export PGHOST=`boot2docker ip 2>/dev/null || echo 127.0.0.1`"
```

That's not perfect, but works for me&trade; on my Mac and on a Linux box without `boot2docker`.